src/platforms/apple/common/dsym.mdx

```diff
@@ -6,17 +6,14 @@ title: Uploading Debug Symbols
 tags: []
 ---
 
-Sentry requires a dSYM upload to symbolicate your crash logs. The symbolication process unscrambles Apple’s crash logs to reveal the function, file names, and line numbers of the crash. Upload the dSYM file using either [sentry-cli](https://github.com/getsentry/sentry-cli), the [Fastlane](https://fastlane.tools/) action, or set up the Sentry [App Store Connect](#bitcode-appstore) integration for Bitcode builds.
+Sentry requires a dSYM upload to symbolicate your crash logs. The symbolication process unscrambles Apple’s crash logs to reveal the function, file names, and line numbers of the crash. Upload the dSYM file using either [sentry-cli](https://github.com/getsentry/sentry-cli), the [Fastlane](https://fastlane.tools/) action, or set up the Sentry [App Store Connect](#bitcode-appstore) integration for Bitcode builds. Every solution requires a Sentry Auth Token with the [correct permissions](/product/cli/dif/#permissions). You can [create one here](https://sentry.io/api/).
 
-## With Bitcode {#dsym-with-bitcode}
+# With Bitcode {#dsym-with-bitcode}
 
-If [Bitcode](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/AppThinning/AppThinning.html#//apple_ref/doc/uid/TP40012582-CH35-SW2) **is enabled** in your project, dSYM files will only be available **after** builds have finished processing in App Store Connect. We recommend using the latest Xcode version for submitting your build.
+If [Bitcode](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/AppThinning/AppThinning.html#//apple_ref/doc/uid/TP40012582-CH35-SW2) **is enabled** in your project, dSYM files will only be available **after** App Store Connect finished processing the build. Sentry can fetch dSYM files automatically as they become available from App Store Connect when setting up the [App Store Connect](#bitcode-appstore) integration.
+Alternatively, the dSYM files can be downloaded either with [Fastlane](#bitcode-fastlane) or manually, and uploaded with [Fastlane](#bitcode-fastlane) or [sentry-cli](#bitcode-sentrycli).
 
-Sentry can fetch dSYM files automatically as they become available from App Store Connect when setting up the [App Store Connect](#bitcode-appstore) integration.
-
-Alternatively, the dSYM files can be downloaded either with [Fastlane](#bitcode-fastlane) or manually, and uploaded with [sentry-cli](#bitcode-sentrycli).
-
-### Using App Store Connect {#bitcode-appstore}
+## Using App Store Connect {#bitcode-appstore}
```

Comment:
My suggested edit here should be *added* in after the note below, so you'll need to make these changes on your branch. 

```suggestion
### Using App Store Connect {#bitcode-appstore}

This section covers the necessary setup for upload methods:
-[Upload BCSymbolMaps Using Fastlane](#symbolmap-fastlane)
-[Upload BCSymbolMaps Using sentry-cli](#symbolmap-sentrycli)
```